selftests/pre_release/tests/vmimage.py

```diff
@@ -65,8 +68,21 @@ def test_get(self):
 
 
 class ImageFunctional(Base):
+
+    def setUp(self):
+        super().setUp()
+        self.cache_list = set(i['file']
+                              for i in vmimage_plug.list_downloaded_images())
+
     @fail_on(process.CmdError)
     def test_get(self):
         cmd = 'avocado vmimage get --distro=%s --distro-version=%s --arch=%s'
         cmd %= (self.vmimage_name, self.vmimage_version, self.vmimage_arch)
         process.run(cmd)
+
+    def tearDown(self):
+        new_cache_files = set(i['file']
+                              for i in vmimage_plug.list_downloaded_images())
```

Comment:
Thank you for adding this inside tearDown! :+1: 

My only suggestion is to remove the duplicated loop, putting it inside a private method and call this method on setUp and tearDown.

Reply:
True, I will change that.